tendrl/node_agent/node_sync/disk_sync.py

```diff
@@ -273,9 +273,7 @@ def get_node_block_devices(disks_map):
     out, err, rc = cmd.run()
     if not err:
         out = out.encode('utf8')
-        devlist = map(
-            lambda line: dict(zip(keys, line.split(' '))),
-            out.splitlines())
+        devlist = [dict(list(zip(keys, line.split(' ')))) for line in out.splitlines()]
```

Comment:
if you could, put a comment on top of this, it would have been good... Otherwise this line looks very cryptic (for any future maintainer)...

From the code, what I understood is,
a. we are running a command `lsblk` and printing specific set of columns (NAME, KNAME, PKNAME etc...) to the output
b. for each line in the output, we are (creating list of dictionaries) mapping the specific column to the specific output field.

Reply:
Sure